integration-tests/templates/_Cargo.toml

```diff
@@ -7,10 +7,14 @@ edition = "2021"
 crate-type = ["cdylib"]
 
 [dependencies]
-near-sdk = { version = "4.1.0-pre.1", features = ["abi"] }
 serde = { version = "1", features = ["derive"] }
 schemars = "0.8"
 
+[dependencies.near-sdk]
+git = "https://github.com/near/near-sdk-rs.git"
+rev = "13fbfc9930fb0ee162dd9c726ce02e15219c956d"
+features = ["abi"]
+
```

Comment:
should probably wait for an sdk prerelease before merging this in, or wdyt?

Reply:
What is missing for us to release 4.1.0-pre.2? Are we waiting on https://github.com/near/near-sdk-rs/pull/891 and https://github.com/near/near-sdk-rs/pull/893 to be pulled in?

Reply:
We can cut the prerelease without those two. Since we still need to properly review the ABI embedding.

Reply:
Ok, I suggest we ask @austinabell to cut 4.1.0-pre.2 and wait for the release then

Reply:
ah, I'll do that now

Reply:
done